lib/mdl/Slider.js

```diff
@@ -188,6 +197,8 @@ class Slider extends Component {
   _onTouchEvent(evt) {
     switch (evt.type) {
       case 'TOUCH_DOWN':
+        this._startTouch(evt);
+        break;
       case 'TOUCH_MOVE':
         this._updateValueByTouch(evt);
```

Comment:
This is a breaking change since `_updateValueByTouch()` is never called again. 

Taking a look at the commit you refer, `_updateValueByTouch()` is called inside the `beginMove` function. Can you add it in your `startTouch` or `emitStart` ?